cmd/world/main.go

```diff
@@ -36,6 +37,12 @@ func main() {
 	// Set logger sentry hook
 	log.Logger = log.Logger.Hook(telemetry.SentryHook{})
 
+	// Set up config directory "~/.worldcli/"
+	err := globalconfig.SetupConfigDir()
+	if err != nil {
+		log.Err(err).Msg("could not setup config folder")
```

Comment:
```suggestion
		log.Err(err).Msg("could not setup config folder")
		return
```
close the program if config dir cannot be setup